sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -219,7 +219,7 @@ If you're using [Azure China](https://docs.microsoft.com/azure/china/china-welco
 azure.activedirectory.environment=cn-v2-graph
```

Comment:
These will be deleted in another PR, right?

Reply:
There will be a PR later to update readme and changelog, but in case I forget this point, let me fix it now.